lms/templates/help_modal.html

```diff
@@ -18,194 +18,207 @@
-            close_time = "21:00"
-          else:
-            open_time = "14:00"
-            close_time = "22:00"
-        %>
-        <p>
-        ${Text(_(
-            'Thank you for your inquiry or feedback.  We typically respond to a request '
-            'within one business day (Monday to Friday, {open_time} UTC to {close_time} UTC.) In the meantime, please '
-            'review our {link_start}detailed FAQs{link_end} where most questions have '
-            'already been answered.'
-        )).format(
-            open_time=open_time,
-            close_time=close_time,
-            link_start=HTML('<a href="{}" target="_blank" id="success-feedback-faq-link">').format(marketing_link('FAQ')),
-            link_end=HTML('</a>')
-        )}
-        </p>
-    </div>
+    <%
+      dst = datetime.now(pytz.utc).astimezone(pytz.timezone("America/New_York")).dst()
+      if dst:
+        open_time = "13:00"
+        close_time = "21:00"
+      else:
+        open_time = "14:00"
+        close_time = "22:00"
+    %>
+    <p>
+    ${Text(_(
+        'Thank you for your inquiry or feedback.  We typically respond to a request '
+        'within one business day (Monday to Friday, {open_time} UTC to {close_time} UTC.) In the meantime, please '
+        'review our {link_start}detailed FAQs{link_end} where most questions have '
+        'already been answered.'
+    )).format(
+        open_time=open_time,
+        close_time=close_time,
+        link_start=HTML('<a href="{}" target="_blank" id="feedback-faq-link" tabindex="0">').format(marketing_link('FAQ')),
+        link_end=HTML('</a>')
+    )}
+    </p>
   </div>
 </section>
 
 <script type="text/javascript">
-$(document).ready(function() {
-    var $helpModal = $("#help-modal"),
-        $closeButton = $("#help-modal .close-modal"),
-        $leanOverlay = $("#lean_overlay"),
-        $feedbackForm = $("#feedback_form"),
-        onModalClose = function() {
-            $closeButton.off("click");
-            $leanOverlay.off("click");
-            $helpModal.attr("aria-hidden", "true");
+(function() {
+    var onModalClose = function() {
+            $("#help-modal .close-modal").off("click");
+            $("#lean_overlay").off("click");
+            $("#help-modal").attr("aria-hidden", "true");
             $('area,input,select,textarea,button').removeAttr('tabindex');
             $(".help-tab a").focus();
-            $leanOverlay.removeAttr('tabindex');
+        },
+        cycle_modal_tab = function(from_element_name, to_element_name) {
+            $(from_element_name).on('keydown', function(e) {
+                var keyCode = e.keyCode || e.which;
+                if (keyCode === 9) {
+                    e.preventDefault();
+                    $(to_element_name).focus();
+                }
+            });
         };
 
-    DialogTabControls.setKeydownListner($helpModal, $closeButton);
-
-    $helpModal.click(function () {
-        $helpModal.focus();
```

Comment:
@cahrens I think it was due to this line, I was out this week will look into it on Monday.